astro/src/content/docs/get-started/use-cases/app-suite.mdx

```diff
@@ -0,0 +1,129 @@
+
+These applications may:
+
+* be hosted at different domains or under different paths
+* use SAML or OIDC for authentication
+* be custom, commercial, or open source
+* be written using different technology stacks
+
+You want to let your users log in once, and switch between applications without having to log in again. Users have different roles for each application.
+
+## Solution
+
+With the app suite use case, after the user logs in to one application, they will transparently log in to every application via the magic sauce of FusionAuth SSO. You'll also have only one customer profile to manage. You can view their login activity across the applications.
+
+## Prerequisites
+
+You have configured your application or applications to [delegate authentication to FusionAuth](/docs/get-started/use-cases/auth-service) via the Authorization Code grant and the FusionAuth hosted login pages.
+
+## Example Scenario
+
+Suppose you have a thriving e-commerce business with multiple applications:
+
+* an e-commerce store where people can buy clown costumes
+* a virtual try-on app which lets you see yourself in the outfits before you buy
+* a forum for discussion about the latest trends
+* a ticketing system for customer support
+
+Here's a diagram of an app suite as a central hub for these different applications.
+
+![Diagram of an app suite as a central hub for a number of different applications.](/img/docs/get-started/use-cases/app-suite.png)
+
+All of these use FusionAuth for their login. Each is represented by a different application and client Id, but your users transparently log in each time they switch between applications.
+
+## Actors/Components
+
+* your user and their client application (mobile app or browser)
+* two or more applications
+* FusionAuth
+
+This use case is built on cookies and redirects to FusionAuth. The redirect is typically transparent to the end user.
+
+## Implementation Steps
+
+This is a two step implementation process to ensure customers can access all the applications. Steps to take:
+
+* Ensure end users have their FusionAuth SSO session enabled when they first authenticate.
+* When a user is not logged in to a particular application, forward the user to FusionAuth. They'll be transparently logged in and delivered back to the application.
+
+### Force SSO Session Creation
+
+The easiest way to force the SSO session to be created as a user logs in is to modify the login page. You do so by modifying the theme. In the default theme, a user can check or uncheck the `rememberDevice` checkbox, labelled by default "Keep me signed in". To force creation of the SSO session, update the theme to set the `rememberDevice` parameter to `true`. Modify the `OAuth Authorize` template [using the advanced theme editor](/docs/customize/look-and-feel/advanced-theme-editor)
+
+Here's a sample FreeMarker block you can use to update the default template.
+
+```
+[@hidden name="rememberDevice" value="true"/]
+```
+
```

Comment:
Do we need a note about Simple themes Folx being SOL for the time being?

Reply:
I don't think so. We talk about the advanced theme editor above.